test/Xpring/xpring-client.test.ts

```diff
@@ -1,14 +1,15 @@
-import { assert } from 'chai'
 import bigInt from 'big-integer'
+import { assert } from 'chai'
 import { FakeWallet } from 'xpring-common-js'
-import FakeXRPClient from '../XRP/fakes/fake-xrp-client'
-import FakeXRPPayIDClient from '../PayID/fakes/fake-xrp-pay-id-client'
-import { testXRPTransaction } from '../XRP/fakes/fake-xrp-protobufs'
-import TransactionStatus from '../../src/XRP/transaction-status'
-import XpringClient from '../../src/Xpring/xpring-client'
-import RawTransactionStatus from '../../src/XRP/raw-transaction-status'
+
 import { XRPLNetwork } from '../../src'
+import XpringClient from '../../src/Xpring/xpring-client'
 import XpringError from '../../src/Xpring/xpring-error'
+import RawTransactionStatus from '../../src/XRP/raw-transaction-status'
+import TransactionStatus from '../../src/XRP/transaction-status'
+import FakeXRPPayIDClient from '../PayID/fakes/fake-xrp-pay-id-client'
+import FakeXRPClient from '../XRP/fakes/fake-xrp-client'
+import { testXRPTransaction } from '../XRP/fakes/fake-xrp-protobufs'
```

Comment:
This should really be in another PR, but it's really just moving around some import statements, and I'm lazy.

Reply:
Looks reasonable to me. If you're doing massive amounts of work to clean up a codebase you can be lazy.